src/js/components/Accordion/stories/Custom.js

```diff
@@ -7,7 +7,12 @@ import { Accordion, AccordionPanel, Box, Grommet, Text } from 'grommet';
 
 const CustomAccordionTheme = {
   accordion: {
-    heading: { level: '3' },
+    heading: {
+      level: 3,
+    },
+    hover: {
+      color: 'accent-2',
```

Comment:
I looked at the storybook example, and I don't see this hover color.

Reply:
Its just on the last one because the first two are not passing a `string` to heading so thats not getting the hover either way.  I just ran it again on mine and it shows if you can just check the last one ? 

![Screen Shot 2020-04-28 at 4 31 04 PM](https://user-images.githubusercontent.com/42451602/80544220-cbb07080-896d-11ea-8c91-61f267709712.png)